docs/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Alert users when Bugzilla sync failed (OSIDB-3252)
+- Record last impact increase in trackers (OSIDB-3448)
```

Comment:
Please rebase changelog. You are updating release 4.3.3

Reply:
The `Unreleased` section is still outdated, compare with current [changelog](https://github.com/RedHatProductSecurity/osidb/blob/master/docs/CHANGELOG.md).